data/tsv/egy_latn_phonemic.tsv

```diff
@@ -3331,7 +3263,6 @@ zꜣṯw	z ɑ t͡ʃ uː
 ꜥnḫ	ʕ aː n a χ
 ꜥnḫ	ʕ aː n ə χ
 ꜥnḫ	ʕ oː n ə χ
-ꜥnḫ.tj	ɑː n x ʔ t i
```

Comment:
Intresting. I don't know much about transliteration schemes for Ancient Egyptian, but "." may indeed stand for something reasonably valid. How do we check?